### DISCUSSION TITLE
Critical functions are missing as well as unused.

### DISCUSSION THREAD

**Discussion by IAm0x996** [solodit/feedback#44] [Bug Report] [answered]
I noticed that Solodit is currently missing the "report_tags" function, and the latest audit report is currently missing the "Category_tags".

![image](https://github.com/solodit/feedback/assets/151486859/122b1126-ac62-4328-aec0-f049ece3d8ee)

![image](https://github.com/solodit/feedback/assets/151486859/eb488d09-b3c8-4570-9443-65385f6948e8)

**Comment by hans-cyfrin** [comment, marked as answer]
The "Report Tag" filter is available in second place from the top.
For the undefined categories and tags, the Solodit team does not handle tagging and categorization. 
They are open to change for all users, and we expect the community to work together to evolve it.
